desktop/apps/buy_art/stylesheets/fonts.styl

```diff
@@ -0,0 +1,169 @@
+@font-face
```

Comment:
You are right, we should be able to just import the existing ones

(I guess we actually didn't use most of it, we might be able to just remove them entirely)